common/changes/miwhea-fabric-core-6_2017-02-07-22-38.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Update to Fabric Core 6.0.0",
+      "type": "patch"
```

Comment:
I'm leaning towards this being a major change. When updating, developers will also need to update their references to Fabric Core 6.0 or there will be breaking changes in the app, such as fonts not being set in components.